catchup/catchpointService.go

```diff
@@ -156,11 +160,16 @@ func MakeNewCatchpointCatchupService(catchpoint string, node CatchpointCatchupNo
 }
 
 // Start starts the catchpoint catchup service ( continue in the process )
-func (cs *CatchpointCatchupService) Start(ctx context.Context) {
+func (cs *CatchpointCatchupService) Start(ctx context.Context) error {
+	err := cs.checkLedgerDownload()
```

Comment:
what if the stage is "catchpoint downloaded" and node was restarted? In this case no need to go into network at all.

Reply:
That's true. I think in the case of a resumed catchpoint service we should just succeed since we assume it was previously successfully started.

Reply:
The handlers invoke [StartCatchup](https://github.com/algorand/go-algorand/blob/master/node/node.go#L1164) via the node which checks that the service isn't already running. So you'd only get a 200 the first time.